Packs/SMIME_Messaging/Integrations/SMIME_Messaging/SMIME_Messaging.yml

```diff
@@ -108,5 +108,5 @@ script:
   type: python
   subtype: python3
 tests:
-- No tests - added unitest
+- No tests
```

Comment:
why did you touch this?

Reply:
to try and handle `CJ105` err. but maybe it is just a dev bug. WDYT?

Reply:
sorry, wrong file my bad.